build.gradle

```
@@ -36,12 +36,12 @@ subprojects {
 
     // artifact properties
     group = 'org.apache.flink'
-    version = '1.11-SNAPSHOT'
+    version = '1.12-SNAPSHOT'
     description = """Flink Training Exercises"""
 
     ext {
         javaVersion = '1.8'
-        flinkVersion = '1.11.2'
+        flinkVersion = '1.12.0'
```

let's increase that to 1.12.2 while we're changing it
```suggestion
        flinkVersion = '1.12.2'
```